lib/runner/variant-reassignment.js

```diff
@@ -86,6 +98,7 @@ export default class VariantReassignment {
         { concurrency: 3 })
       return this.errorCallback(error, productDraft)
     }
+    this.statistics.retries++
     const transactions = await this.transactionService.getTransactions()
     const failedTransaction = transactions.find(({ value }) =>
         _.isEqual(value.newProductDraft.name, productDraft.name)
```

Comment:
Why match transactions by product name where product name is not unique? Why not slug for example?

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/commit/e72a6f5cf4350a509ab9710fae9f284d1981d680